apps/i18n/netsim/en_ploc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "yes": "Yes"
```

Comment:
There is a "common" loc file that gets used as well - no automatic inheritance, just manual.  So yes, this particular string should probably go there.

Do I even need to create this file yet?  I'm not totally clear on what pseudoloc does.